packages/gatsby-theme-carbon/src/components/AnchorLinks/AnchorLinks.module.scss

```diff
@@ -52,18 +47,18 @@
 }
 
 // dark background
-:global(.container--dark) .link {
-  color: $carbon--white-0;
+// :global(.container--dark) .link {
+//   color: $carbon--white-0;
 
-  &:before {
-    color: $carbon--white-0;
-  }
+//   &:before {
+//     color: $carbon--white-0;
+//   }
 
-  &:hover {
-    color: $carbon--blue-40;
+//   &:hover {
+//     color: $carbon--blue-40;
 
-    &::before {
-      color: $carbon--blue-40;
-    }
-  }
-}
+//     &::before {
+//       color: $carbon--blue-40;
+//     }
+//   }
+// }
```

Comment:
Is this part still WIP or did you mean to delete the comments?

Reply:
Thx!